demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -95,3 +97,22 @@ def is_html_doc(self) -> bool:
             return True
         # use some heuristics to try to figure out if this is html
         return txt.startswith('<p>') or ('<thead>' in txt and '<tbody>' in txt)
+
+    def is_image_path_valid(self) -> bool:
+        with open(self.file_path) as f:
+            readme_content = f.read()
+            invalid_paths = re.findall(
+                r'(\!\[.*?\]|src\=)((\(|\")https://github.com/demisto/content/(?!raw).*?(\)|\"))', readme_content,
+                re.IGNORECASE)
+            if invalid_paths:
+                for path in invalid_paths:
+                    # Grabbing only url and removing " " or ( ) url wrapper
+                    path = path[1][1:-1]
+                    alternative_path = path.replace('blob', 'raw')
+                    print_error(f'============ Detected following image url: ============\n{path} \n'
```

Comment:
please attach a screenshot of the error

Reply:
<img width="1211" alt="Screen Shot 2020-08-06 at 11 10 38" src="https://user-images.githubusercontent.com/21142167/89507683-8524aa80-d7d5-11ea-9913-f8f42d880db0.png">

Reply:
remove the `============` wrapper